modules/simple-payments/simple-payments.css

```diff
@@ -16,6 +16,7 @@
 .jetpack-simple-payments-price {
```

Comment:
Maybe you have some suggestion here @iamtakashi, using flexbox for instance. Feel free to edit this file.